manager/integration/tests/common.py

```diff
@@ -4558,3 +4558,17 @@ def restore_backup_and_get_data_checksum(client, core_api, backup, pod,
                                      'default')
 
     return data_checksum, output, restore_pod_name
+
+def restart_k3s_agent():
+    cmd = ['service k3s-agent restart']
+    subprocess.check_call(cmd)
+
+def check_k3s_agent_log(keyword, minutes=60*3):
+    # Check the keyword in k3s-agent log for x minutes every second of the output
+    cmd = ['journalctl -u k3s-agent | grep -i ' + keyword]
+    for _ in range(minutes*60):
```

Comment:
We have common retry range variables like `RETRY_COMMAND_COUNT` etc

Reply:
Hey Khushboo, I think what I trying to do here is to tailing the log after waiting for certain amount of time. I will see what I can weak this function with **RETRY_COMMAND_COUNT**.